src/js/popup.js

```diff
@@ -131,8 +131,7 @@ function send_error(message) {
     var origins = badger.getAllOriginsForTab(tabId);
     if(!origins){ return; }
     var version = chrome.runtime.getManifest().version;
-    //TODO "there's got to be a better way!"
-    var fqdn = tab.url.split("/",3)[2];
```

Comment:
This change looks good, although it won't fix any of the NULL fqdn field error report scenarios I pasted earlier since none of them are http URLs; `about:` URLs will keep resolving to `""`. That's OK though, not an important issue.